pydantic/main.py

```diff
@@ -133,6 +133,7 @@ class BaseModel(metaclass=_model_construction.ModelMetaclass):
         # pydantic._internal._generate_schema.GenerateSchema.model_schema to work for a plain BaseModel annotation
         model_fields = {}
         __pydantic_decorators__ = _decorators.DecoratorInfos()
+        __pydantic_parent_namespace__ = None
```

Comment:
I noticed this attribute could fail to be defined in some places where it was accessed. Explicitly setting it to `None` here fixed that.